notion_client/api_endpoints.py

```diff
@@ -325,3 +325,29 @@ def list(self, **kwargs: Any) -> SyncAsync[Any]:
             query=pick(kwargs, "block_id", "start_cursor", "page_size"),
             auth=kwargs.get("auth"),
         )
+
+
+class OAuthEndpoint(Endpoint):
+    def introspect(self, **kwargs: Any) -> SyncAsync[Any]:
+        """Introspect the provided token.
```

Comment:
Nit: Let's stay close to the Notion docs wording.

```suggestion
        """Get the token's active status, scope, and issued time.
```